pureport_client/commands/__init__.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, client=None):
         :returns: an instance of CommandBase
         :rtype: `pureport_client.commands.CommandBase`
         """
+        client.make_bindings()
```

Comment:
instead of calling `make_bindings()` here i would suggest that you add `automake_bindings=True` here --> https://github.com/pureport/pureport-python-client/blob/devel/pureport_client/__main__.py#L74